audbcards/core/datacard.py

```diff
@@ -114,7 +114,8 @@ def example_media(self) -> typing.Optional[str]:
         )
         # Download of example data might fail
         try:
-            media = self.dataset.deps.media[index]
+            files = audb.info.files(self.dataset.name, version=self.dataset.version)
+            media = files[index]
```

Comment:
This was the only place inside `audbcards.Datacard` in which we needed the `audb.Dependencies` object to be present. I removed it and replaced it by calling `audb.info.files()` instead. This is not ideal, and we might consider storing the `example_media` attribute as part of `audbcards.Dataset`. Or we cache the media file inside the cache, and call `audb.info.files()` only when the file cannot be found in the cache. But as this is related to `audbcards.Datacard`, I would like to tackle it in another pull request.

Reply:
This change is no longer needed